infra/testing/webdriver/windowLoaded.js

```diff
@@ -17,8 +17,7 @@ const windowLoaded = async () => {
   await executeAsyncAndCatch(async (cb) => {
     const loaded = () => {
       if (!window.Workbox) {
-        const error = new Error('Workbox not yet loaded...');
-        cb({error: error.stack});
+        cb({error: `window.Workbox is undefined; location is ${location.href}`});
```

Comment:
This is what helped me realize that when the `workbox-window` tests were failing, it was because a tab navigated to the `workbox-broadcast-update` URL had the focus...